api/CSPViolationReportBody.json

```diff
@@ -11,7 +11,7 @@
           "chrome_android": "mirror",
           "edge": "mirror",
           "firefox": {
-            "version_added": false
+            "version_added": "63"
```

Comment:
How did you get this version number?  Did you use test code, or a bug?

Reply:
Based on 

* https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/63#apis
* https://bugzilla.mozilla.org/show_bug.cgi?id=1472661

Reply:
Bug 1472661 seems to refer to the `SecurityPolicyViolationEvent` API rather than this one (`CSPViolationReportBody`).  I don't see any IDL for this interface in Firefox either.

I think that the file you actually want to modify is `http/headers/Content-Security-Policy.json`, since that's specific to the HTTP header?

Reply:
I was also thinking that, but there are two parts to this topic:

* HTTP headers which set policies (`Content-Security-Policy: default-src 'self'; report-uri http://mycoolurl.com/reports`)
* Violation reports -> JSON payloads the browser will send to a configurable URL if a policy in the page headers are violated:

```json
{
  "csp-report": {
    "document-uri": "http://mycoolurl.com/somepage.html",
    "referrer": "",
    "blocked-uri": "http://example.com/css/style.css",
    "violated-directive": "style-src",
    "original-policy": "default-src 'self'; report-uri http://mycoolurl.com/reports"
  }
}
```